tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -64,11 +63,11 @@
 #
 # Let's take a look:
 
-mne.viz.plot_alignment(raw.info, trans=trans, subject='sample',
-                       subjects_dir=subjects_dir, surfaces='head-dense',
-                       show_axes=True, dig=True, eeg=[], meg='sensors',
-                       coord_frame='meg')
-mlab.view(45, 90, distance=0.6, focalpoint=(0., 0., 0.))
+fig = mne.viz.plot_alignment(raw.info, trans=trans, subject='sample',
+                             subjects_dir=subjects_dir, surfaces='head-dense',
+                             show_axes=True, dig=True, eeg=[], meg='sensors',
+                             coord_frame='meg')
+mne.viz_3d_view(fig, 45, 90, distance=0.6, focalpoint=(0., 0., 0.))
```

Comment:
Here it's `mne.viz.set_3d_view()`. That's why [circle](https://circleci.com/gh/mne-tools/mne-python/13866?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) is complaining.